client.go

```diff
@@ -226,7 +212,7 @@ func (client *Client) marshallJSON(data interface{}) (string, error) {
 	return string(response), err
 }
 
-func (client *Client) unMarshallDataToJson(stream io.Reader, result interface{}) error {
+func (r *marathonClient) unMarshallDataToJSON(stream io.Reader, result interface{}) error {
```

Comment:
I think it should be `unmarshalDataToJSON` (one `l`, one word) according to this https://golang.org/pkg/encoding/json/#Unmarshal